src/extension/migration.js

```diff
@@ -0,0 +1,41 @@
+export default function migrateStyle(properties) {
+  const dataTemplateLabel = {
+    label: {
+      value: {
+        colorType: undefined,
+        color: undefined,
+        colorExpression: undefined,
+      },
+    },
+  };
+
+  const dataTemplateBackground = {
+    backgroundColor: {
+      colorType: undefined,
+      color: undefined,
+      colorExpression: undefined,
+    },
+  };
+
+  const dataTemplateBorder = {
+    border: {
+      top: undefined,
+      fullBorder: undefined,
+      colorType: undefined,
+      color: undefined,
+      colorExpression: undefined,
+    },
+  };
+
+  if (!properties.components || properties.components.length === 0) {
+    Object.assign(dataTemplateLabel.label.value, properties.style?.fontColor);
+    Object.assign(dataTemplateBackground.backgroundColor, properties.style?.backgroundColor);
+    Object.assign(dataTemplateBorder.border, properties.style?.border);
+    const labelComponent = { key: "label", ...dataTemplateLabel };
```

Comment:
Do you really need to create another object here just to assign the key? Why not juse add the key when `dataTemplateLabel` is created?